pkg/common/connectionmanager/connectionmanager.go

```diff
@@ -188,3 +188,17 @@ func (cm *ConnectionManager) VerifyWithContext(ctx context.Context) error {
 	}
 	return nil
 }
+
+func (cm *ConnectionManager) APIVersion(vcenter string) (string, error) {
```

Comment:
can you add a comment here? to keep `go vet` happy 😄

Reply:
Will do